root/layout/components/BottomMenu.js

```diff
@@ -136,6 +136,9 @@ component AboutMenu() {
         <li>
           <a href="https://metabrainz.org/gdpr">{l('GDPR compliance')}</a>
         </li>
+        <li>
+          <a href="/doc/Copyright_DMCA">{l('Copyright / DMCA Issues')}</a>
```

Comment:
```suggestion
          <a href="/doc/Copyright_DMCA">{l('Copyright / DMCA issues')}</a>
```

We use sentence case nowadays